packages/core/src/geometry/Geometry.ts

```diff
@@ -31,12 +31,10 @@ const map: Dict<any> = {
  *
  * Geometry can be defined without passing in a style or data if required (thats how I prefer!)
  * @example
- * import { Geometry } from 'pixi.js';
+ * const geometry = new PIXI.Geometry();
```

Comment:
We like to use import for examples instead of the global PIXI. Majority of use-cases are using bundles like Webpack.